python/housinginsights/ingestion/Cleaners.py

```diff
@@ -308,6 +308,21 @@ def replace_tracts(self,row,row_num,column_name='census_tract'):
             logging.warning('  no matching Tract found for row {}'.format(row_num,row))
         return row
 
+    def append_tract_label(self,row,row_num,column_name='census_tract_number'):
```

Comment:
Thanks!  This was about as far as I could stretch my amateur Python skills, but I was really excited to discover that it worked when I looked at the records in my local copy of the db.